### PR TITLE
[1] Refactor: restructure repo and renaming files

### PR DESCRIPTION
**Pull request by anniero98**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
- Moved all helper files to `lib` and named them `featureUtils.js` (thanks to people power for the inspiration)
- Renamed all the `featureDetailedScreen`s to `featureDetailsScreen`
- Deleted `AppNavigator.web.js` because it was .. odd and redundant. Couldn't find any info on the web
- Renamed all the `Announcement` -related files to be `News` (as in the Figma)
- Cleaned up some small things (poor naming) etc
- Renamed and merged `stores` and `products` folders to `map`, and updated the appropriate filenames as well

- Updated the generator version (and updated all the files)
- Created folders for all `components`; note that unlike the `screens` folder, I created both a `store` and `product` folder. For `screens` it made more sense to put them all in `map`; for `components`, however, there are too many `product` and `store` subcomponents so it seems to make sense to separate those concerns.
- Changed the name of the `styled-components` folder from `styles` to `styled` to alleviate some confusion on `styled-components` vs. `components` in the sense of being contained and reusable combinations of elements

- Lastly, small changes on many of the files (needed to fix the imports)

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### Related PRs
Branches off of #23 

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review
Check out the new layout (moved all helper functions to `lib`) and please give feedback if you think there are better names for various files or functions. It does seem semantic but if we're going to change things, we should make any changes that might be good now.

[//]: # "The order in which to review files and what to expect when testing locally"

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

~Planning on adding `babel-plugin-inline-dotenv` to our app to manage the environment variables (https://stackoverflow.com/questions/57060316/environment-variables-in-babel-config-jsv)~ (Update: Added in #23 !); setup the integration with the updated schema generator code (https://github.com/aivantg/airtable-schema-generator). And finally, refactor the Airtable code (may make some changes to the generated code. TBD on branch `annie/airtable`). Might be a fat PR. Might not.

## Tests Performed, Edge Cases
Everything should work as before - hopefully. I've run through all the usual flows to make sure navigation is working as expected even with the renaming and affected imports.

CC: @wangannie

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

**Comment by JustinMi** [comment]
Hey Annie! I'm a bit busy this week so I don't have the bandwidth to catch up on the codebase and give this PR the proper review it deserves, but I should be freer starting next week. If this PR is urgent, I think it's best to ask someone else to review it; I want to make sure that I'm not the blocker for you here.

**Comment by anniero98** [comment]
@JustinMi No worries! For some context, believe we're aiming to have two reviewers per PR from now on. 

Tagging in @ethanlee16 @FranCogMents @aivantg @kyuwonk  (though this one is blocked on #23 ) if any of y'all have time since these are both relatively big architectural changes and wanted to get feedback from you! cc @wangannie

File: screens/auth/SignUpScreen.js [hunk 1]
```diff
@@ -166,7 +168,13 @@ export default class SignUp extends React.Component {
...
     let formattedPhoneNumber = this.state.phoneNumber;
     // eslint-disable-next-line prettier/prettier
-    formattedPhoneNumber = `(${formattedPhoneNumber.slice(0, 3)}) ${formattedPhoneNumber.slice(3, 6)}-${formattedPhoneNumber.slice(6, 10)}`;
+    formattedPhoneNumber = `(${formattedPhoneNumber.slice(
+      0,
+      3
+    )}) ${formattedPhoneNumber.slice(3, 6)}-${formattedPhoneNumber.slice(
+      6,
+      10
+    )}`;
```
Comment (thumn) [inline screens/auth/SignUpScreen.js]: This is a result of prettifying the .slice method right :0 this happens to me too (unsure why they think this is cleaner??)

File: lib/newsUtils.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+function createNewsItemData(newsItem) {
+  return {
+    date: new Date(newsItem.get('Created')),
+    description: newsItem.get('Description'),
+    title: newsItem.get('Title'),
+    id: newsItem.get('ID')
+  };
+}
+
+const getNewsItems = function async() {
```
Comment (aivantg) [inline lib/newsUtils.js]: Maybe add a TODO if you want to ideally replace this to use something from a future version of the schema generator? Just so it's easy to go back to this

File: lib/newsUtils.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  return BASE('News')
+    .select({
+      view: 'Grid view',
+      sort: [{ field: 'Created', direction: 'desc' }]
```
Comment (aivantg) [inline lib/newsUtils.js]: Also, realizing this now...you could just use the `getAllNews()` function and sort it locally?

File: screens/auth/SignUpScreen.js (hunk 1 above)
Comment (aivantg) [reply screens/auth/SignUpScreen.js]: yeah HAHA this is so weird. Maybe you could outsource it to a small helper function just so it's easier to look at? 😂

File: screens/map/MapScreen.js [hunk 4]
```diff
@@ -9,19 +9,24 @@ import {
...
   View,
-  Button
-, Dimensions } from 'react-native';
+  Button,
+  Dimensions
+} from 'react-native';
 import MapView, { Marker } from 'react-native-maps';
 import BottomSheet from 'reanimated-bottom-sheet';
-import Hamburger from '../../components/Hamburger'
+import Hamburger from '../../components/Hamburger';
```
Comment (aivantg) [inline screens/map/MapScreen.js]: I love that this is a component

File: lib/newsUtils.js (hunk 3 above)
Comment (anniero98) [reply lib/newsUtils.js]: This is a great point (-: I still think it'd be nice to have optional sort functions but it doesn't matter in the grand scheme of things 😂

File: lib/newsUtils.js (hunk 2 above)
Comment (anniero98) [reply lib/newsUtils.js]: Addressing on the new branch (`annie/airtable`)

**Pull request merged**